vnext/ReactUWP/Modules/Animated/NativeAnimatedModule.cpp

```diff
@@ -0,0 +1,198 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include "pch.h"
+#include "NativeAnimatedModule.h"
+
+#include <winrt/Windows.ApplicationModel.DataTransfer.h>
+#include "unicode.h"
+#include <cxxreact/Instance.h>
+#include <cxxreact/JsArgumentHelpers.h>
+
+//#pragma warning(push)
+//#pragma warning(disable : 4146)
+//#include <cxxreact/JsArgumentHelpers.h>
+//#pragma warning(pop)
+
+#if _MSC_VER <= 1913
```

Comment:
don't need to disable optimize if you're not using any co_await stuff in the file